packages/preview/ustc-proposal/0.0.1/typst.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "ustc-proposal"
+version = "0.0.1"
+entrypoint = "lib.typ"
+authors = ["Wu, Zhenyu"]
+license = "GPL-3.0"
+description = "中国科学技术大学开题报告的 Typst 模板"
```

Comment:
Could you please add an English description too? I am personally not sure what the package is for which makes it harder to review.